Chapters/4-8-Chapter30.md

```diff
@@ -0,0 +1,285 @@
+
+"Where is Annu?"
+
+Fanla and Setre paused, puzzled at the question, but Boon raised her head to look in a specific direction and let out two "woop" sounds. A silhouette with soft blue eyes slowly appeared behind Setre, still bound with a rope leading to Setre. Thi was safe for now.
+
+Setre leaned in close and asked me loud enough to hear, "Who did this?"
+
+I responded with the name she least wanted to hear.
+
+"Ramne."
+
+She instinctively turned her head to Annu but I gently pulled her hand to bring her closer.
+
+I said to her sternly, "I'll never forget what you almost did," and sank back down to wait for the ringing in my ears to pass. Fanla took me in her arms and moved me toward the dying fire as Setre stood in the dark, spanning the distance of her unrest, keeping an eye on Annu as thi hesitantly joined Fanla and me.
+
+A crowd slowly began to form, the laborers returning from their work, alarmed at the sound of an explosion. They huddled around Setre, asking questions and fearing for their lives at each answer. Annu sat near Fanla as she massaged my neck to relieve the ringing sound. Thi stared blankly into the darkness.
+
+"You're looking for him," I said to hae. Fanla was startled by the volume of my voice.
+
+"Listening," thi said loudly back to me. "I cannot see."
+
+Fanla chimed in, "Ramne won't be back with this many people. He only strikes when he sees an opportunity."
+
+"How do you know?" I asked.
+
+"All of the ruin you see around you was caused by him. The smoke that slipped you into the great darkness - that was caused by him. Nobody knew how it had started until long after it had ravaged this place."
+
+I told them, "He coerced me because I was alone."
+
+"Then it's best the three of us stick together from now on," she said.
+
+Fanla stopped massaging and shook her hand to loosen it up. I motioned for her to stop when she tried to go back to my neck.
+
+"It's fine. I'm fine," I said, sitting up. She moved her hands to her trousers, dusting off some invisible debris before placing them together on her lap. But before she could settle in, a clamor of people erupted into the camp as blue lights flailed about and small explosions crackled through the trees. The three of us leapt to our feet as the explosions grew closer and closer.
+
+"Holragi!" the guardians yelled. "Holragi!"
+
+Setre flung herself toward the commotion and found some cover behind a big ziyu. She covered the light at her chest and arms with her poncho, and stooped low to stay hidden. I saw Fanla start to follow her but she stopped abruptly, seeing something moving in the distance. Immediately, she turned around and signalled for us to run.
+
+I yelled to Annu, "It's time."
+
+Thi nodded and bellowed to the crowd "To Ulu! Run to Ulu!"
```

Comment:
wait aren't they already in ulu?

Reply:
i think you need to do more of that thing you do really well. set the scene, so readers know where they are.